code/projects/app/src/app/stages/e1/e1.component.ts

```diff
@@ -39,7 +38,9 @@ export class E1Component {
 
     this.textContent = extractText(image);
     this.textSubject.next(this.textContent);
-    this.textBound = Math.floor(image.bytes.length / 7);
+    this.textBound = Math.floor(
+      (image.imageData.width * image.imageData.height * 3) / 7
```

Comment:
Ah yes, the magic numbers. I hope that in the reworked version by @mlodybercik will not have them.